pkg/sidecar/sidecar.go

```diff
@@ -88,7 +98,16 @@ func Run(version string) int {
 		return 1
 	}
 
-	// run until terminated
+	// run grpc server until terminated
+	server, err := grpc.NewServer(rt.KubeClient, grpc.ServerConfig{
```

Comment:
Given that we're going to use the GRPC `healthz` package, we should probably start the grpc server before calling `rt.WaitTillCSIDriverIsValidated`, so that we can change the health state to `SERVING` after the call succeeds.